docs/en/ingest-management/release-notes/release-notes-8.4.asciidoc

```diff
@@ -27,6 +27,47 @@ Also see:
 [[release-notes-8.4.1]]
 == {fleet} and {agent} 8.4.1
 
+[discrete]
+[[known-issues-8.4.1]]
+=== Known issues
+
+// tag::credentials-error[]
+.Credentials error prevents {agent} from ingesting AWS logs
+[%collapsible]
+====
+
+*Details*
+
+{agent}s configured to use the AWS S3 and SQS integrations may return the
```

Comment:
I would be "less" specific: meaning is a problem related to any AWS integration and not specific to S3 and SQS.
It also involves Crowdstrike, Cisco Umbrella and AWS Logs integrations.